src/localLighthouse.js

```diff
@@ -160,11 +157,38 @@ export default async ({
       overrides,
       throttling,
       throttlingMethod,
-      url
-    });
+      url,
+      verbose
+    };
 
-    auditResults.push(lighthouseAuditResult);
-    index++;
+    if (options.emulatedFormFactor !== 'both') {
+      if (verbose) {
```

Comment:
Sorry if I wasn't clear @Mirzamohammad22. My point is that we don't need to have this block repeated now. We can leave it where it was.

Reply:
No issues , let me know  if the `If case` is fine or should i revert to lighthouseCaller